dragster.js

```diff
@@ -755,12 +755,16 @@
                     placeholderPosition = dropTarget.getAttribute(placeholderAttrName);
 
                 if (placeholderPosition === POS_TOP) {
-                    insertBefore(dropDraggableTarget, dropTemp);
+                    insertBefore(dropDraggableTarget, finalParams.wrapDraggableElements === FALSE ? draggedElement : dropTemp);
                 } else {
-                    insertAfter(dropDraggableTarget, dropTemp);
+                    if (finalParams.wrapDraggableElements === FALSE) {
+                        insertAfter(draggedElement, dropDraggableTarget);
+                    } else {
+                        insertAfter(dropDraggableTarget, dropTemp);
+                    }
                 }
 
-                if (draggedElement.firstChild) {
+                if (draggedElement.firstChild && finalParams.wrapDraggableElements !== FALSE) {
```

Comment:
`finalParams.wrapDraggableElements === TRUE` here we want to get a `true` value instead of any truthy value.